core/admin/admin.php

```diff
@@ -883,6 +883,7 @@ private function get_experiments() {
 					'state' => $experiment_data['state'],
 					'default' => $experiment_data['default'],
 					'dependencies' => $dependencies,
+					'messages' => $experiment_data['messages'] ?? [],
```

Comment:
```suggestion
					'messages' => $experiment_data['messages'] ?? (object) [],
```
check if it works, and also change the mocks if it does

Reply:
Changed to object.